go.mod

```diff
@@ -28,25 +25,24 @@ require (
 	github.com/operator-framework/operator-lib v0.6.0
 	github.com/operator-framework/operator-lifecycle-manager v0.17.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.46.0
-	github.com/prometheus/client_golang v1.11.0
+	github.com/prometheus/client_golang v1.11.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/rook/rook v1.8.3
+	github.com/rook/rook v1.9.9
```

Comment:
The 4.10 branch must be based on v1.8, not on v1.9

Reply:
so how we'll get API from rook based on 1.9.9?